src/js/components/uploadDetachedFiles/UploadDetachedFileValidation.jsx

```diff
@@ -213,7 +213,7 @@ export default class UploadDetachedFileValidation extends React.Component {
 		submission.meta['endDate'] = this.state.rep_end;
 		submission.meta['subTierAgency'] = this.state.agency;
 
-		this.uploadFileHelper(kGlobalConstants.LOCAL === true && !this.isUnmounted, submission)
+		this.uploadFileHelper(kGlobalConstants.LOCAL === true, submission)
```

Comment:
I was so focused on the other half of this that I didn't realize this other detail. You don't have to do `kGlobalKonstants.LOCAL === true` you can just do `kGlobalConstants.LOCAL` because it's either true or false. No need to check when you're basically checking what the boolean is.